Changed Chracters instance modal to check between different relations

swindiggie opened the pull request:
Changed Characters instance modal to check between different relations for one that actually has data.

So if a character is not associated with any anime or voice actors, then it will just display manga.

For all instances, if one relation doesn't have any data, then no title is shown and the chart of the other relation takes on full-width

Commit:
Changed Chracters instance modal to check between different relationships for one that actually has data

dmalone7 reviewed (commented):
If a character is from both an anime and manga, is it supposed to show both associations?  Spike Spiegel is in both an anime and a manga (as far as I know) and it only shows anime/VO for him.

Commit:
Merge branch 'master' into check-relation-data

Commit:
Changed lists in the modal to be passed in as an array. Character modals now contain 3 different lists.

Commit:
Fixed overflow on the subinfo of a thumbnail card. Also made an empty modal non-white so it doesn't flash at the user while it is loading.

dmalone7 reviewed (commented):
Titles on the Manga modal are missing sometime, and the formatting of text under the card is odd.

https://i.imgur.com/lnCXdgH.jpg

dmalone7 reviewed (approved):
Issues fixed, looks nice, good for approval.
